hedyorg/hedy
# 🪲 Fix highlighting of keywords with spaces

To highlight words that have spaces in them, we actually replace the spaces with `|` and use a regex to match the keyword. The problem arose there, since I used a string and not a regex, which caused only the first occurrence of the space to be replaced 😵 

Fixes #5189

**How to test**

I added a test in Spanish, but you can also go to level 15, set the language to Spanish and paste this code:

```
l = 1, 2, 3
si 4 not in l
```

And it should be highlighted correctlty!
